src/screens/HomeScreen/HowYouFeel.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {
+  Text,
+  View,
+  AsyncStorage,
+  StyleSheet,
+  FlatList,
+  Image,
+  Dimensions,
+  TouchableOpacity,
+} from 'react-native';
+
+import neutral from '@/assets/images/Emoji/neutral.png';
+import happy from '@/assets/images/Emoji/happy.png';
+import cry from '@/assets/images/Emoji/cry.png';
+import fearful from '@/assets/images/Emoji/fearful.png';
+import love_hearts from '@/assets/images/Emoji/love_hearts.png';
+import tired_face from '@/assets/images/Emoji/tired_face.png';
+import unamused from '@/assets/images/Emoji/unamused.png';
+import disappointed from '@/assets/images/Emoji/disappointed.png';
+
+const HowYouFeel = () => {
+  const emoticons = [neutral, happy, cry, fearful, love_hearts, tired_face, unamused, disappointed];
+
+  saveData = async item => {
+    const emotionList = [
```

Comment:
No need to define this again. you can use `emoticons` for this.

Reply:
Hello @narender2031 basically `emoticons` are the imported images. I created `emotionList` object because we need to save the **name of the emotion** as selected by the user. If we use `emoticons` then the name of the emotion won't get saved since it is an imported image.